Source/Core/Core/HW/WiimoteEmu/Speaker.cpp

```diff
@@ -106,18 +106,17 @@ void Wiimote::SpeakerData(wm_speaker_data* sd)
 		return;
 	}
 
-	// Speaker Pan
-	unsigned int vol = (unsigned int)(m_options->settings[4]->GetValue() * 100);
-
+	// Speaker Pan using Constant Power Pan Law
+	double pan = m_options->settings[4]->GetValue() / 1.27;
+	double pan_prime = 3.14159265358979323846 * (pan + 1) / 4.0;
```

Comment:
yea, I just checked. If cmath was included earlier in the project without _USE_MATH_DEFINES, it's not possible reinclude it with the defines enabled (however with math.h is works fine). The other solution is for _USE_MATH_DEFINES to be set globally. So I guess either, use math.h instead of cmath, use global define, or something lioncash suggested.